### PR TITLE
Minor changes in README docs

### PR DESCRIPTION
**Pull request by olemis**
Changes:
- Spelling corrections in README
- Initial project CHANGELOG

Does this change need to mentioned in CHANGELOG.md?

No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Security: in case of vulnerabilities.
+
+Dates must be YEAR-MONTH-DAY
+-->
+
+## Unreleased (0.1.0-alpha) - 2019-01-30
+
+### Added
+
+- Explanatory document about the build process: [Build_Skybian.md](Build_Skybian.md)
```
Comment (olemis) [inline CHANGELOG.md]: @stdevPavelmc @some4 please add in CHANGELOG all other things implemented in this repository for Skybian images .

File: .travis.yml [hunk 2]
```diff
@@ -18,10 +18,11 @@ deploy:
   api_key:
     secure: iqmt0/JGzw+QXQwth/gla1M6KA0aOgYuCQEBjaRMNmtLAJn96Nn6qWGvbubrrjlcEPGNC1S6r2fx1MkPz+MvoOpGWohQ3Jxr+5+kARmNa2GlxuaLJpH5VNqqpVswGWrqrwOYV5NzdrPxnq4dXX8rFaeHAL1XykFzIxLQqadzr1inBknuhSY6vaqCawXLC+2+H7J4wvGSyw6o3G0LGBEljAxYeBVRHuDXOnP3XfooRQLw/brqIm9d1ZGhaBqNN22qiBqCuvEjvFGQf27YCzIad6oyU0zdsvvauZRKzosCBqEW353Ww6HOIrAaZW98U6GdWkFAqzB4vDznrQUu9ybiDyQTIuK/yYrA7kGM5R2Vi7ZGoR6qatF005vZhqjXQP5GH7TcTCOiZeYyyR91cF66Bf7Olxjs0UloiBYZWNUZ7E99VkJ7S/8hiYr+epSjdHdF9KmLY7GTp7iBq0aV89MCgz6LNYBd3Yoh1Ll/72WYwomaeAAg32K6Z7yFPym+3lMQ4h7SY4QAmHrQKr/qD3gJZTlBWsl/i/R8+6lcRZ5napSJJiS3knToEJMs9Y3N+grk04C3Xtigy3to7XLXSIseVYa2aLf1DtPKi3Nti2IHjOvYhR9pU5hfo272zNb3bfpBLLoMR4FqFquphPYT/X0Saakv7zQ3/EoUEcdcUqOMIqg=
   file: output/final/*.tar.xz
+  file_glob: true
   skip_cleanup: true
   overwrite: true
   on:
-    repo: simelo/skybian
-    branch: release
+    all_branches: true
```
Comment (olemis) [inline .travis.yml]: Not related to PR description . In a separate PR pls

File: build.sh [hunk 3]
```diff
@@ -2,20 +2,40 @@
 
 # This is the main script to build the Skybian OS for Skycoin miners.
 #
-# Author: stdevPavelmc@github.com, @co7wt in telegram
-# Skycoin / Simelo teams
+# Author: stdevPavelmc@github.com, @pavelmc in telegram
+# Skycoin / Simelo team
```
Comment (olemis) [inline build.sh]: Just Skycoin

File: build.sh [hunk 4]
```diff
@@ -2,20 +2,40 @@
...
-. ${ROOT}/environment.txt
+. ${ROOT}/build.conf
 
 ##############################################################################
 # This bash file is structured as functions with specific tasks, to see the
 # tasks flow and comments go to bottom of the file and look for the 'main'
 # function to see how they integrate to do the  whole job.
 ##############################################################################
 
+# Capturing arguments to show help
```
Comment (olemis) [inline build.sh]: Not related to PR description . Submit in another one please.

**Comment by stdevPavelmc** [comment]
The purpose of this PR is obsolete, as all the issues has been covered in other side issues. There is only one commit that needs attention: https://github.com/skycoin/skybian/pull/8/commits/3a01fccb4f92cdedccd34d02eeac2bc53381762f, @some4 please make a fetch upstream against current skycoin/skybian and re-send your PR, sorry for the inconvenience.

So I will close the issue.

**Pull request closed without merging**